core/client/fs/src/main/java/alluxio/client/file/policy/LocalFirstAvoidEvictionPolicy.java

```diff
@@ -37,62 +30,24 @@
 @ThreadSafe
 public final class LocalFirstAvoidEvictionPolicy
```

Comment:
Is this class still here just for backward compatibility? Should we deprecate it?

Reply:
backwards compatibility is a part, but we also don't have a great way to pass arguments such as `boolean avoidEviction` to location policies.